### PR TITLE
13205: Remove ___from_store if Add Store Code to Url set to Yes

### PR DESCRIPTION
**Pull request by jakubderdziak**
### Fix 13205
- add ___from_store only if "Add Store Code to Urls" is set to "No"

### Fixed Issues (if relevant)

1. magento/magento2#13205:  In magento 2.2.2 Url Options issue

### Manual testing scenarios (*)
1. Enable Store switcher
2. Set Add Store Code to Urls to Yes
3. While switching stores, there should not be __from_store param in URL
4. Set Add Store Code to Urls to No
5. While switching stores, there should be __from_store param in URL

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/magento2?pullRequest=18693) <br/>All committers have signed the CLA.

**Comment by magento-engcom-team** [comment]
Hi @jakubderdziak. Thank you for your contribution 
Here is some useful tips how you can test your changes using Magento test environment.
Add the comment under your pull request to deploy test or vanilla Magento instance:
- `@magento-engcom-team give me test instance` - deploy test instance based on PR changes
- `@magento-engcom-team give me $VERSION instance` - deploy vanilla Magento instance

For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation

**Comment by sidolov** [comment]
Hi @jakubderdziak  , looks like you made some commits with email different than in your GitHub profile, please, add email from commits to your profile!
Thank you!

**Comment by jakubderdziak** [comment]
@sidolov Corrected. Sorry for mistake.

**Review by orlangur — CHANGES_REQUESTED** [review]
Nowadays all changes must be applied to `2.3-develop` first (see https://devdocs.magento.com/guides/v2.3/contributor-guide/contributing.html#rules). Please prepare a new pull request, this one should be `on hold` until the latter is merged.

**Comment by orlangur** [comment]
Closing due to inactivity. Feel free to reach me out anytime later if you wish to continue work on this pull request and it will be reopened.
Thanks for collaboration!

**Pull request closed without merging**
